crates/relayer/src/chain/mock.rs

```diff
@@ -471,6 +490,34 @@ impl ChainEndpoint for MockChain {
     ) -> Result<(AnyConsensusState, MerkleProof), Error> {
         unimplemented!()
     }
+
+    fn cross_chain_query(
```

Comment:
didn't use gRPC right?

Reply:
sry, it mislead, let me correct.
the result of query defined in the module is not stored in blockchain.
It's evaluated at runtime and it wasn't proved by any validators, so that module receiving result from relayer cannot validate the value is valid or not.

Reply:
Thank you the idea of potential failure of system.
I'll try to find query path that can be proofed.